### DISCUSSION TITLE
How to use ory cli to connect to local ory hydra?

### DISCUSSION THREAD

**Discussion by baominwang** [ory/hydra#3591] [Q&A]
If we are using ory cli, it would send request to Ory consoler. I didn't find a parameter to specify the endpoint for connecting to local Ory hydra. How?

**Comment by vinckr** [comment]
Hello @baominwang 

When you are self-hosting Ory Hydra, you have to use the built-in `hydra` CLI, for example: https://www.ory.sh/docs/hydra/cli/hydra-create-client

This is already installed as part of Hydra, so you dont need the Ory CLI - that is just for Ory Network.
